components/Dropdown/index.js

```diff
@@ -0,0 +1,35 @@
+// function myFunction() {
+//     document.getElementById('myDropdown').classList.toggle('show')
```

Comment:
Nice thought process here! I would have also probably used the toggle method as well.

Reply:
Also, thank you for commenting out your prior work! I love seeing you progress and make changes. 🔥